File: CS/WinForms_NeutralFilterMode_Example/Form1.cs
using DevExpress.DashboardWin;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace WinForms_NeutralFilterMode_Example
{
    public partial class Form1 : DevExpress.XtraEditors.XtraForm
    {
        public Form1()
        {
            InitializeComponent();
            dashboardViewer1.UseNeutralFilterMode = true;
            dashboardViewer1.CustomizeDashboardTitle += DashboardViewer1_CustomizeDashboardTitle;
            dashboardViewer1.UpdateDashboardTitle();
            dashboardViewer1.DashboardSource = typeof(Dashboard1);
        }

        private void DashboardViewer1_CustomizeDashboardTitle(object sender, DevExpress.DashboardWin.CustomizeDashboardTitleEventArgs e)
        {
            DashboardViewer dashboardViewer = (DashboardViewer)sender;

            DashboardToolbarItem commandButton = new DashboardToolbarItem(dashboardViewer.UseNeutralFilterMode, "NEUTRAL FILTER MODE",
                new Action<DashboardToolbarItemClickEventArgs>((args) => {
                    dashboardViewer.UseNeutralFilterMode = !dashboardViewer.UseNeutralFilterMode;
                    dashboardViewer1.DashboardSource = typeof(Dashboard1);
                }));
            commandButton.Caption = "NEUTRAL FILTER MODE";
            e.Items.Add(commandButton);
        }
    }
}

File: VB/WinForms_NeutralFilterMode_Example/Form1.vb
Imports DevExpress.DashboardWin
Imports System
Imports System.ComponentModel
Imports System.Drawing

Namespace WinForms_NeutralFilterMode_Example

    Public Partial Class Form1
        Inherits DevExpress.XtraEditors.XtraForm

        Public Sub New()
            InitializeComponent()
            dashboardViewer1.UseNeutralFilterMode = True
            AddHandler dashboardViewer1.CustomizeDashboardTitle, AddressOf DashboardViewer1_CustomizeDashboardTitle
            dashboardViewer1.UpdateDashboardTitle()
            dashboardViewer1.DashboardSource = GetType(Dashboard1)
        End Sub

        Private Sub DashboardViewer1_CustomizeDashboardTitle(ByVal sender As Object, ByVal e As CustomizeDashboardTitleEventArgs)
            Dim dashboardViewer As DashboardViewer = CType(sender, DashboardViewer)
            Dim commandButton As DashboardToolbarItem = New DashboardToolbarItem(dashboardViewer.UseNeutralFilterMode, "NEUTRAL FILTER MODE", New Action(Of DashboardToolbarItemClickEventArgs)(Sub(args)
                dashboardViewer.UseNeutralFilterMode = Not dashboardViewer.UseNeutralFilterMode
                dashboardViewer1.DashboardSource = GetType(Dashboard1)
            End Sub))
            commandButton.Caption = "NEUTRAL FILTER MODE"
            e.Items.Add(commandButton)
        End Sub
    End Class
End Namespace
